perfkitbenchmarker/linux_packages/mongodb_server.py

```diff
@@ -18,6 +18,7 @@
 
 def YumInstall(vm):
   """Installs the mongodb package on the VM."""
+  vm.RemoteCommand('sudo setenforce 0')
```

Comment:
> My Question... Can you turn this back on after the install and does MongoDB continue to run properly?

SELinux prevents the starting of MongoDB and not the installation.  I am unsure at this time if I set SELinux back to enforcing after successfully starting MongoDB if it will continue to run.  I can grab the audit logs and see exactly what is blocked from the SELinux side to get that resolved the right way.